openmrs/i18n/registration/locale_fr.json

```diff
@@ -27,7 +27,9 @@
   "REGISTRATION_INSTITUTE_NAME_TEXT_KEY": "MINSANTE",
   "REGISTRATION_INSTITUTE_REGISTERED_TEXT_KEY": "(Enregistré)",
   "REGISTRATION_INSTITUTE_ADDRESS": "Hopital de la Cité Verte, District - Yaoundé",
-  "REGISTRATION_EMERGENCY_CONTACT_KEY": "No. Tel Personne à prévenir",
+  "REGISTRATION_EMERGENCY_CONTACT_NAME_KEY": "Emergency contact name",
```

Comment:
@prashantsable1991 Can you please get this translations from Lisa?

Reply:
@lauravignoli I already ask Lisa about missing translations but she did not reply for that.